RobotCrawler/src/main/java/server/service/BookFinderController.java

```diff
@@ -0,0 +1,37 @@
+package server.service;
+
+import model.BookData;
+import model.EBookCategory;
+import model.response.EBooksResponse;
+import model.response.ResponseBase;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import robot.ebooks.EBooksRobotManager;
+
+import java.util.EnumMap;
+import java.util.List;
+
+@RestController
+@RequestMapping("/robot")
+public class BookFinderController {
+
+    @Autowired
+    @Qualifier("ebooks-robot-manager")
```

Comment:
Actually I thought that using Qualifier helps Spring to not looking literally for types, name :) 
Thank you for the advice I will fix it:)